src/libnegf.F90

```diff
@@ -876,11 +898,8 @@ subroutine set_mpi_bare_comm(negf, mpicomm)
     type(Tnegf), intent(inout) :: negf
```

Comment:
A comment on this bit (nothing to be done). I was adding this to get a way to pass a communicator for the python API from mpi4py and libnegf_API. That strategy probably needs to be looked at again, but I'd say as a separate PR.